activerecord/lib/active_record/railties/databases.rake

```diff
@@ -295,24 +295,24 @@ db_namespace = namespace :db do
     seed = false
 
     ActiveRecord::Base.configurations.configs_for(env_name: ActiveRecord::Tasks::DatabaseTasks.env).each do |db_config|
-      ActiveRecord::Base.establish_connection(db_config.config)
+      if ActiveRecord::Base.database_exists?(db_config)
+        ActiveRecord::Tasks::DatabaseTasks.migrate
 
-      ActiveRecord::Tasks::DatabaseTasks.migrate
+        # Skipped when no database
```

Comment:
I guess this comment can now be deleted. Previously there was an error happening line above, so the flow was not obvious by looking at the code, but now it is quite straightforward.

Reply:
👍